Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/base_mail.yml

```diff
@@ -95,7 +95,7 @@
         Blunt: 10
   - type: CargoSellBlacklist
   - type: Food # Frontier - Moth food
-    requiresSpecialDigestion: true
+  #  requiresSpecialDigestion: true
   - type: SolutionContainerManager
     solutions:
       food:
```

Comment:
Dont mess with this one, we have a conflicted rework

Reply:
Can do, just thought you'd get a laugh out of it!